src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageFormDetails.tsx

```diff
@@ -77,6 +81,12 @@ const WorkPackageEditDetails: React.FC<Props> = ({
             />
           </FormControl>
         </Grid>
+        <Grid item xs={12} md={2} lg={2} sx={{ mt: 2, mr: 2 }}>
```

Comment:
the way grid works is that there are 12 slots available in a row, so make sure to use them all up (especially in a form like this). I'd make this one md={3} and remove the lg option and do the same thing for the stage select grid item. That should allow the inputs to take up more of the screen